### ISSUE TITLE
Lack of acute response to NO challenge

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6503]
**Preferred term label:**  
Lack of acute response to NO challenge 
**Synonyms**
**Definition (free text, please give PubMed ID)** 
Lack of adequate haemodynamic response to NO challenge (adequate response is defined as a  reduction of mean pulmonary artery pressure (mPAP)  ≥ 10 mmHg to reach an absolute value of mPAP ≤ 40 mmHg with an increased or unchanged cardiac output (CO) in response to inhaled nitric oxide at 10-20 ppm) PMID: 15939821
**Parent term (use hpo.jax.org/app)**
Pulmonary arterial hypertension HP:0002092 
**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**
Idiopathic pulmonary arterial hypertension ORPHA:275766
**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
Note that HPO terms are framed in terms of Abnormalities, and if this is an expected response, it would not be a good HPO term. The HPO term would be somethign like "Lack of acute response to ccb"

**Comment by eswietlik** [comment]
Thank you, Peter, I may need some help with this. I have changed the term to "Lack of acute response to NO challenge" but I would also like to have a term that describes the opposite situation as well. Both conditions (response and lack of response) are pathological as they describe a variable reaction in patients with PAH.

**Comment by pnrobinson** [comment]
@eswietlik My question would be how a patient without PAH would respond to the NO challenge -- i.e., what is a normal response to the challenge? It is hard to encode say the fact that a patient with PAH has a certain normal phenotype is actually an abnormal finding. In this case, probably it is better to say Inappropriately normal response to ... or something like that, let's discuss.

**Comment by eswietlik** [comment]
We would never do the test in someone without pulmonary hypertension; such individuals would be unlikely to respond (fulfil the criteria) as it would mean that their mean pulmonary pressure would drop to zero. Response or lack of thereof are the features of idiopathic pulmonary arterial hypertension which impact on the treatment modality we use and long term outcome.

**Comment by eswietlik** [comment]
As with the other term, I would follow your suggestion change the label to Pulmonary arterial hypertension with the lack of  acute response to NO challenge

**Comment by pnrobinson** [comment]
added with modification as with https://github.com/obophenotype/human-phenotype-ontology/issues/6508
